apps/src/templates/manageStudents/ManageStudentsFamilyNameCell.jsx

```diff
@@ -1,61 +1,53 @@
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
-import {connect} from 'react-redux';
+import React from 'react';
+import {useDispatch} from 'react-redux';
 import ReactTooltip from 'react-tooltip';
 import _ from 'lodash';
 import i18n from '@cdo/locale';
 import {editStudent} from './manageStudentsRedux';
 import {tableLayoutStyles} from '../tables/tableConstants';
 
-class ManageStudentFamilyNameCell extends Component {
-  static propTypes = {
-    id: PropTypes.number.isRequired,
-    familyName: PropTypes.string,
-    isEditing: PropTypes.bool,
-    editedValue: PropTypes.string,
-    sectionId: PropTypes.number,
-    inputDisabled: PropTypes.bool,
+function ManageStudentFamilyNameCell({
```

Comment:
nit: I tend to prefer this way of declaring the default export because it's in the same spot where you're first declaring the function.

```suggestion
export default function ManageStudentFamilyNameCell({
```

Reply:
Good callout. Updated!